File: PhaoHoaDemoPublic/GRAPHICS/Java/PhaoHoaDemo03/src/phaohoademo03/Global.java
package phaohoademo03;

import java.awt.Color;
import java.awt.geom.Point2D;

public class Global {
    public static final int WindowWidth = 700;
    public static final int WindowHeight = 500;
    
    public static Point2D.Double GiaToc = new Point2D.Double(0, 0.15);
    
    public static Color[] ListColors = {
        Color.WHITE, Color.RED, Color.GREEN, Color.BLUE,
        Color.PINK, Color.MAGENTA, Color.YELLOW,
        new Color(0xF0FFFF), new Color(0xFFFAF0), new Color(0xE0FFFF),
        new Color(0xFFD700), new Color(0xFAFAD2)
    };
}

File: PhaoHoaDemoPublic/GRAPHICS/Csharp/PhaoHoaDemo03/Global.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;


namespace PhaoHoaDemo03
{
    class Global
    {
        public static int WindowWidth;
        public static int WindowHeight;

        public static PointF GiaToc = new PointF(0, 0.15f);

        public static readonly Brush[] DsCoVe = { Brushes.White, Brushes.Red, Brushes.Green, Brushes.Blue,
                                                Brushes.Yellow, Brushes.Orange, Brushes.Violet,
                                                Brushes.LightCyan, Brushes.LightPink, Brushes.Snow,
                                                Brushes.FloralWhite, Brushes.Linen, Brushes.LemonChiffon,
                                                Brushes.Azure, Brushes.SpringGreen, Brushes.LawnGreen,
                                                Brushes.GreenYellow, Brushes.PaleTurquoise, Brushes.Khaki,
                                                Brushes.Orchid, Brushes.Plum, Brushes.Thistle, Brushes.Honeydew};
    }
}
